src/cmd/builtin.py

```diff
@@ -98,7 +98,7 @@ async def _restart(self, cmd_line: List[str], execution_ctx: ExecutionContext) -
         await Command.send_message(execution_ctx, f"{execution_ctx.message_author()} invoked restarting the bot!")
         subprocess.call([sys.executable, "walbot.py", "restart"])
 
-    async def _version(self, cmd_line: List[str], execution_ctx: ExecutionContext) -> None:
+    async def _version(self, cmd_line: List[str], execution_ctx: ExecutionContext) -> str:
```

Comment:
In fact, it doesn't return 'str' always, it can be None sometimes. Please, use Optional[str] here. This comment is applied to all cases where you have added ' -> str'

Reply:
Done.